packages/flutter/test/material/search_test.dart

```diff
@@ -1155,6 +1155,73 @@ void main() {
     await tester.pump();
     expect(textField.controller!.text.length, 15);
   }, skip: kIsWeb); // [intended] We do not use Flutter-rendered context menu on the Web.
+
+  testWidgets('Block entering text on disabled widget', (WidgetTester tester) async {
```

Comment:
Just noticed these tests are added in the wrong test file. Tests for `SearchAnchor`/`SearchBar` should be put in `search_anchor_test.dart`. So sorry for missing this.

Reply:
fixed

Reply:
Thanks! I will look for a second review once all tests have passed:)